core_java/twitter/src/main/java/ca/jrvs/apps/twitter/example/appleJson

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
remove if this file is not used..
twitter/src/main/java directory should contain java files only